src/columnsTypes/types/Label/Component.jsx

```diff
@@ -127,6 +116,9 @@ export const LabelsChooser = ({
           label.title.toLowerCase().includes(inputValue.toLowerCase())
         );
 
+  const isNotExists = (label) =>
+    tableLabels.filter(({title}) => title === label).length === 0;
```

Comment:
let's use [includes](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/includes)

Reply:
includes  does a `obj1 === obj2` comparison it will not work. i will change it to [some](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/some)

Reply:
[find](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/find)